modules/sdk/src/main/scala/org/tessellation/sdk/package.scala

```diff
@@ -45,6 +46,9 @@ package object sdk {
       classOf[IDN] -> 528,
       classOf[ConsensusPeerDeclaration[_, _]] -> 529,
       EventTrigger.getClass -> 530,
-      TimeTrigger.getClass -> 531
+      TimeTrigger.getClass -> 531,
+      classOf[OrdinalTrustMap] -> 532,
```

Comment:
These registrations are not needed now

Reply:
The `kryo` registrations have been removed.